packages/app/src/app/pages/Dashboard/index.tsx

```diff
@@ -88,7 +88,6 @@ export const Dashboard: FunctionComponent = () => {
                   />
                 ) : (
                   <Element
-                    as="aside"
```

Comment:
this was creating two `aside` tags nested, <Sidebar> is already adding the content as an `aside`